template/typescript/package.json

```diff
@@ -7,15 +7,19 @@
   "repository": "{{repo}}",
   "main": "dist/index.js",
   "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
   "jsnext:main": "dist/index.es.js",
+  "files": [
+    "dist"
+  ],
```

Comment:
Currently, you're publishing the whole code, but we need to publish only the output files.
Here I set the `dist` to be outputted (alongside the default `package.json` and `README.md`).
In the `.npmignore` I removed the `*.test.*` files from the package.

Reply:
As I described in https://github.com/Hermanya/create-react-hook/issues/22 the problem here is that when I publish create react hook, only the first folder will upload to npm. I'm not sure how to differentiate between the npm publish I do and the npm publish the users of this tool will run on packages created.

Reply:
OK, I removed this.